first/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+
+    <nav class="NavBar">
+
+        <p><span>BR</span> Architect</p>
+        <ol>
+            <li><a href="#"> Project</a></li>
+            <li><a href="#"> About</a></li>
+            <li><a href="#"> Contact</a></li>
+        </ol>
+    </nav>
+
+    <header id="Header">
+
+        <img src="./assets/images/5.jpg" alt="">
+        <p class="Text-Photo"><span>BR</span> Architect</p>
+        <p class="mobile-apper"><span> Br</span></p>
+    </header>
+
+    <section class="Projectsheder">
+        <p>Projects</p>
+
+        <div class="ProjectsList">
+
+            <div class="project">
+                <p>summer house</p>
+                <div class="ProjectImage">
+                    <img src="./assets/images/1.jpg" alt="">
```

Comment:
not formatted well, 
align alt under class,
class=""
alt="
apply it for whole document